src/lib/ui/thermostat/ClimateModeMenu.tsx

```diff
@@ -0,0 +1,70 @@
+import { ChevronDownIcon } from 'lib/icons/ChevronDown.js'
+import { OffIcon } from 'lib/icons/Off.js'
+import { ThermostatCoolIcon } from 'lib/icons/ThermostatCool.js'
+import { ThermostatHeatIcon } from 'lib/icons/ThermostatHeat.js'
+import { ThermostatHeatCoolIcon } from 'lib/icons/ThermostatHeatCool.js'
+import { Menu } from 'lib/ui/Menu/Menu.js'
+import { ThermoModeMenuOption } from 'lib/ui/thermostat/ThermoModeMenuOption.js'
+
+const modes = ['heat', 'cool', 'heatcool', 'off'] as const
```

Comment:
Can we use the HvacModeSetting type from the sdk? Something like
```suggestion
const modes: HvacModeSetting[] = ['heat', 'cool', 'heat_cool', 'off'] as const
```